ci/setup_centos6_python37.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# Python 3.6 stands in our way -- nuking it
+yum erase -y rh-python36
+rm -rf /opt/rh/rh-python36
+
+yum install -y gcc bzip2-devel libffi-devel zlib-devel
+
+cd /usr/src/
+
+# pip3.7 needs new openssl
+curl -O -L https://github.com/openssl/openssl/archive/OpenSSL_1_1_1c.tar.gz
+tar -zxvf OpenSSL_1_1_1c.tar.gz
+cd openssl-OpenSSL_1_1_1c
+./Configure shared zlib linux-x86_64
+make
```

Comment:
Can you change this to `make -sj4` to speed it up? Using more cores obviously helps, but all the output turns out to slow things down, so `-s` also helps.